flutter/android/src/main/kotlin/io/sentry/flutter/SentryFlutterPlugin.kt

```diff
@@ -169,7 +169,7 @@ class SentryFlutterPlugin : FlutterPlugin, MethodCallHandler, ActivityAware {
         val version = sdk["version"] as? String
         if (name != null && version != null) {
           val sdkVersion = SdkVersion(name, version)
-          options.setSentryClientName(name)
+          options.setSentryClientName(name + "/" + version)
```

Comment:
```suggestion
          options.setSentryClientName("$name/$version")
```

Good catch, you can use string interpolation in Kotlin.

Reply:
Thanks.